src/db/db.ts

```diff
@@ -0,0 +1,81 @@
+import { Observable, of } from 'rxjs';
+import { Sequelize, ModelCtor } from 'sequelize';
+import { CexOrder, CexOrderInstance } from './cex-order';
+import { CexFee, CexFeeInstance } from './cex-fee';
+import { CexTrade, CexTradeInstance } from './cex-trade';
+import { Logger } from '../logger';
+import { mergeMap } from 'rxjs/operators';
+
+type InitDBparams = {
+  dataDir?: string;
+  logger: Logger;
+};
+
+type InitDBResponse = {
+  CexOrder: ModelCtor<CexOrderInstance>;
+  CexFee: ModelCtor<CexFeeInstance>;
+  CexTrade: ModelCtor<CexTradeInstance>;
+};
+
+const createModels = (sequelize: Sequelize): InitDBResponse => {
+  const models = {
+    CexOrder: CexOrder(sequelize),
+    CexFee: CexFee(sequelize),
+    CexTrade: CexTrade(sequelize),
+  };
+
+  models.CexOrder.hasMany(models.CexTrade, {
+    as: 'cexOrderTrades',
+    foreignKey: 'orderId',
+    constraints: true,
+  });
+  models.CexTrade.belongsTo(models.CexOrder, {
+    as: 'cexOrderInstance',
+    constraints: true,
+    foreignKey: 'orderId',
+  });
+
+  models.CexFee.belongsTo(models.CexOrder, {
```

Comment:
Why is `CexFee` linked to both `CexOrder` and `CexTrade`?

Reply:
Uhm, actually this's from ccxt, it's in both Trade and Order

Reply:
I'm removing fee from Trade nonetheless and keeping it for Order only.